cluster.go

```diff
@@ -61,6 +61,20 @@ func (c *ClusterClient) Close() error {
 	return nil
 }
 
+// PoolLen returns the number of active connections in the pool, for instrumentation
```

Comment:
So it all goes back to the fact that `ClusterOptions` accept a `PoolSize` which is actually per-client and not a per-cluster-client. Returning the `PoolLen()` is supposed to signal the user that the `PoolSize` setting is probably too low, therefore using the maximum seems the right approach to me, I will update the comment.